src/main/java/ulcrs/models/session/Session.java

```diff
@@ -2,53 +2,38 @@
 
 import ulcrs.models.schedule.Schedule;
 
-import java.time.LocalDateTime;
+import java.util.Date;
```

Comment:
I would recommend not using the `java.util.Date` class if possible, as it's generally not recommended (https://stackoverflow.com/questions/36639154/convert-java-util-date-to-what-java-time-type). 

I think the `java.time` class, or Joda Time, which the `java.time` package is based off of, would work in our case.

Reply:
Changed back to LocalDateTime